ci_tools/json_pytest_output.py

```diff
@@ -70,6 +70,9 @@ def     mini_md_summary(title, outcome, failed_tests):
         outcome = values[1] if len(values) >= 2 else None
         out_file = values[2] if len(values) >= 3 else None
 
+
```

Comment:
Is one empty line not enough?

Reply:
Wasn't paying attention. Fixed 06344de36de77bf0f661581417167847a9707a0b